### DISCUSSION TITLE
Not working for js or json

### DISCUSSION THREAD

**Discussion by OBLIXi-ON** [Bekaboo/dropbar.nvim#167] [General]
I've tsserver and json-ls installed but dropbar doesn't show anything. Any idea how to fix this?

**Comment by Bekaboo** [comment]
Please open a bug report with minimal steps & config to reproduce

**Reply by OBLIXi-ON** [reply]
Just figured it out. I had to add my required filetypes in `opts.general.enable`. Now it's working fine. Still don't know how it does automatically detect python but not the other types.
